README_JP6.md

```diff
@@ -160,6 +161,8 @@ sudo /opt/nvidia/jetson-io/config-by-hardware.py -n 2="Jetson RealSense Camera D
 
 # Enable d4xx autoload:
```

Comment:
is the section `## Deploy build results on Jetson target` a repetition of `## Install kernel drivers, extra modules and device-tree to Jetson AGX Orin` section?

Reply:
There is a section for advanced users and newbies with step-by-step commands.